tests/framework/installer/ceph_installer.go

```diff
@@ -776,13 +776,16 @@ spec:
                       #
                       udevadm trigger || true
                       vgimport -a || true
+                      set +o pipefail
```

Comment:
The `pipefail` option is set in this script so as not to mask failures earlier in pipes. This should not be disabled.

For matching VGs to `ceph-` in a safer way idiomatic to Bash, you can use
```bash
for vg in $(...); do
  if [[ $vg =~ ceph- ]]; then
    // stuff
  fi
done
```

Something similar can be done for PVs.

Reply:
I found that `+o pipefail` can be eliminated by using `awk` instead of `grep`. 

```
PVS=$(pvs --noheadings --separator=' ' -o pv_name,vg_name | awk '/ceph-/ {print $1}')
...
for vg in $(vgs --noheadings --readonly --separator=' ' -o vg_name | awk '/ceph-/ {print}'); do
```

I use this way since it's simpler than bash's regexp in this case.

Reply:
@BlaineEXE As a result of @leseb's comment, the latest PR is based on your suggestion, rather than the above-mentioned my idea.